server/src/integrations/environment/environment.validation.ts

```diff
@@ -38,34 +39,44 @@ export class EnvironmentVariables {
   @IsUrl({ require_tld: false })
   FRONT_AUTH_CALLBACK_URL: string;
 
-  @IsString()
+  @Transform(({ value }) => value.toLowerCase() === 'true')
```

Comment:
Ideally, I would like the environment validation to throw if the provided value is not either `'true'`/`'TRUE'`/`'1'` or `'false'`/`'FALSE'`/`'0'` and then convert it to a boolean. I'm unsure how to do that properly. 
I'm thinking about a pre-process of environment variables maybe a little bit more sophisticated than `@Transform` or `enableImplicitConversion`